backend/src/team-eventsAPI.ts

```diff
@@ -3,11 +3,10 @@ import { TeamEvent } from './DataTypes';
 import { PermissionError } from './errors';
 import PermissionsManager from './permissions';
 
-export const getAllTeamEvents = async (user: IdolMember): Promise<TeamEvent[]> => {
-  const canCreateTeamEvent = await PermissionsManager.canEditTeamEvent(user);
-  if (!canCreateTeamEvent) throw new PermissionError('does not have permissions');
-  return TeamEventsDao.getAllTeamEvents();
-};
+export const getAllTeamEvents = (user: IdolMember): Promise<TeamEvent[]> =>
+  // const canCreateTeamEvent = await PermissionsManager.canEditTeamEvent(user);
+  // if (!canCreateTeamEvent) throw new PermissionError('does not have permissions');
```

Comment:
I think some TeamEventAPI tests should work now, so maybe comment these back in?

Reply:
done! thanks for reminding me!